requirements.txt

```diff
@@ -1 +1,2 @@
--e git+https://github.com/poldracklab/niworkflows.git@1c00a2490b7c5c9fd2c5d9afa7b6861d4ccecd08#egg=niworkflows
+-e git+https://github.com/effigies/nipype.git@a5fe6bb#egg=nipype
+-e git+https://github.com/effigies/niworkflows.git@7051b39#egg=niworkflows
```

Comment:
Is it okay to have these pins go to my user? I figure we should probably have these issues resolved by the time we make another release, but in the short term, these are both needed.

Nipype pin: nipype/master version string is `0.13.0-dev` < `0.13.0-rc1` in `info.py`. This pin just updates to `rc2`.
Niworkflows pin: The nipype pin includes nipy/nipype#1784, which was also not in `0.13.0-rc1`. This makes niworkflows compatible with the above pin.

The simplest resolution for both is to have another release candidate in nipype, and let both depend on `>=0.13.0-rc2`.